stack.py

```diff
@@ -1,49 +1,99 @@
-# Make Python 2 use float division instead of integer division
+# -*- coding: utf-8 -*-
 from __future__ import division
+import re
 
 
-# TODO: This is pretty stringly typed, perhaps return a custom object?
-def next_instruction(program):
-    split_program = program.split(";")
-    for instuction in split_program:
-        if instuction == "":
-            raise StopIteration
+class Instr(object):
+    sigil_to_op = {
```

Comment:
In this case, I felt that it would be better to simply not assign sigils in cases where it's not obvious what the best to use would be, so that we can spend more time adding it later. It's perfectly valid to use the words and sigils in the same program, so I don't think this is the time to worry about it.